source/2020-06-19-the-ember-times-issue-153.md

```diff
@@ -0,0 +1,202 @@
+
+The configuration files generated when you first [run the script](https://github.com/atej/ember-new-plus#quickstart-) can be tuned to your preference later. If you are about to make a new Ember project, give [ember-new-plus](https://github.com/atej/ember-new-plus) a try!
+
+---
+
+## [RFC: Type Stability for Addons 💠](https://github.com/typed-ember/ember-cli-typescript/pull/1158)
+
+[Chris Krycho (@chriskrycho)](https://github.com/chriskrycho) proposed a set of guidelines and tooling recommendations for managing changes as addons adopt [TypeScript](https://www.typescriptlang.org/) throughout the Ember ecosystem, as part of the path to making TypeScript a first-class citizen in Ember as a whole.
+
+In the [RFC](https://github.com/typed-ember/ember-cli-typescript/pull/1158), it proposes to align with [Ember’s SemVer and LTS commitments](https://emberjs.com/releases/) and design workflows to support that policy, so that consumers of Ember addons which publish types are insulated from breaking changes in TypeScript whenever possible. The RFC outlines what needs to happen when we encounter:
+
+- Breaking changes to type definitions
+- Non-breaking changes to type definitions
+- Bug fixes to type definitions
+- Dropping support for previously-supported versions
+
+Read through the full [RFC on Type Stability for Addons](https://github.com/typed-ember/ember-cli-typescript/pull/1158) and leave some feedback!
+
+---
+
+## [Stories of becoming a developer 📖](https://dev.to/agathebadia/share-your-dev-story-51dg)
+
+[Agathe Badia (@honeypotio)](https://github.com/Agathebadia) has written a blog post called **Share your Dev story**. This blog post introduces her new website 🎉 that was built with [Ember.js](https://emberjs.com/). It is called [Becoming a Developer](https://becoming-a-developer.netlify.app/).
+
+On Becoming a Developer, you can share your background, the story of how you started, interest in coding, studies, first developer role, and some recommendations for anyone starting or continuing to dive into development. 💻
+
+In her [Share your Dev story](https://dev.to/agathebadia/share-your-dev-story-51dg) post, Agathe talks about how meaningful her new website is to her. You can see her passion as a developer shine. 🌟
+
+Each developer's story is **unique** and **insightful** and can inspire _you_ to do something you never thought possible. [Check it out](https://becoming-a-developer.netlify.app/) and be sure to share your own! 😃
+
+---
+
+## [Farewell Ember A11y Strike Team, hello Working Group for Digital Accessibility 🚀](https://github.com/ember-a11y/core-notes/blob/ember-a11y/ember-a11y/2020-06/june-17.md)
+
+In case you didn't know, the Ember A11y Strike Team will have its final meeting next week 😢 ... but fear not, some of the ✨ amazing ✨ work that the Strike Team did will be continued in the Working Group for Digital Accessibility! 🔥 
```

Comment:
```suggestion
In case you didn't know, the Ember A11y Strike Team, formed to address the [issue that outlined the technical accessibility issues in new Ember apps](https://github.com/emberjs/rfcs/issues/595) will have its final meeting next week 😢 ... but fear not, some of the ✨ amazing ✨ work that the Strike Team did will be continued in the Working Group for Digital Accessibility! 🔥 
```